Questie-BCC.toc

```diff
@@ -211,6 +211,7 @@ Modules\Journey\tabs\QuestsByZone\QuestsByZoneTab.lua
 #Modules\Journey\tabs\Search\SearchTab.lua
 Modules\Journey\QuestieSearch.lua
 Modules\Journey\QuestieSearchResults.lua
+Bindings.xml
```

Comment:
This is incorrect for all TOC files. The `Bindings.xml` file is always loaded without explicitly adding it to the TOC.